sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -1069,6 +1069,23 @@ def dropna(self, axis, **kwargs):
             preserves_partition_by=partitionings.Singleton(),
             requires_partition_by=requires_partition_by))
 
+
+  # TODO(BEAM-XXXX): support @<local> in eval and query. Currently fails in
+  # proxy generation
```

Comment:
Currently if the user specifies a local variable in expr with `@<local>` this will fail at construction time (when generating the proxy), but the error message isn't helpful, it just indicates `<local>` doesn't exist.

Reply:
(I'll update the XXX with a jira number before merging if this gets approved, I just didn't want to create it prematurely)

Reply:
Yeah I'll add a regex to check for `@<py identifier>` and point to the jira.

Reply:
Done.